storage/Utils/SystemCmd.h

```diff
@@ -234,10 +239,12 @@ namespace storage
 	bool _doThrow;
 	OutputProcessor* _outputProc;
 	struct pollfd _pfds[3];
+	string mockup_key;
 
```

Comment:
Why does not this attribute start with underscore as the rest? Is there any criteria?